how-to/maintain/tiup-operations.md

```diff
@@ -97,7 +97,7 @@ tiup cluster display ${cluster-name}
                 log.slow-threshold: 300
         ```
 
-    参数的格式参考 [TiUP 配置参数模版](https://github.com/pingcap-incubator/tiup-cluster/blob/master/topology.example.yaml)。
+    参数的格式参考 [TiUP 配置参数模版](https://github.com/pingcap-incubator/tiup-cluster/tree/master/examples)。
```

Comment:
The link is not wrong. See https://github.com/pingcap/docs-cn/pull/2915/files. Do you find that this link is broken? @CaitinChen

Reply:
@yikeke Thanks. Fixed at https://github.com/pingcap/docs-cn/pull/2935